packages/beta/src/__tests__/api/alertsApi.int.spec.ts

```diff
@@ -201,3 +201,29 @@ test('sort alerts', async () => {
   });
   expect(response.items.length).toBeGreaterThan(0);
 });
+
+test('cursor pagination', async () => {
+  const client: CogniteClient = setupLoggedInClient();
+  const response = await client.alerts.list({
+    limit: 1,
+    sort: {
+      property: 'createdTime',
+      order: 'desc',
+    },
+    cursor: '',
+  });
+  expect(response.items.length).toBe(1);
+
+  // test for more than 1000 items
+  const alerts = client.alerts
+    .list({
+      sort: {
+        property: 'createdTime',
+        order: 'desc',
+      },
+      cursor: response?.nextCursor,
+    })
+    .autoPagingToArray({ limit: 1001 });
+
+  expect((await alerts).length).toBeGreaterThan(0);
```

Comment:
question(non-blocking): is there way to test the number of alerts to be greater than 1000 ?

Reply:
I suppose i could do this: `expect(alerts.length).toBeGreaterThan(1000);`

Reply:
Yes. That'd make sense.
But you might have to make sure that there are more than 1000 alerts present.. So mabye create 1001 alerts and then delete them after the test would be helpful ? 🤔

Reply:
I agree. I do so in my latest commit @saagar-cognite 👍🏻